src/python_testing/TC_VALCC_4_4.py

```diff
@@ -60,53 +68,114 @@ async def test_TC_VALCC_4_4(self):
         self.step(1)
         attributes = Clusters.ValveConfigurationAndControl.Attributes
 
-        self.step(2)
+        self.step("2a")
+        feature_map = await self.read_valcc_attribute_expect_success(endpoint=endpoint, attribute=attributes.FeatureMap)
+
+        is_ts_feature_supported = feature_map & Clusters.ValveConfigurationAndControl.Bitmaps.Feature.kTimeSync
+
+        self.step("2b")
+        if not is_ts_feature_supported:
+            logging.info("TimeSync feature not supported skipping test case")
+
+            # Skipping all remainig steps
```

Comment:
https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/matter_testing_support.py#L1017

Reply:
This will be added in a follow up PR.